src/shared/Actions.ts

```diff
@@ -1,17 +1,30 @@
 import { action } from 'mobx';
-import _ from 'lodash';
-import utils from '../utils';
-import parser from '../parser';
+import * as _ from 'lodash';
+import * as utils from '../utils';
+import * as parser from '../parser';
+import { CommonProperties } from './CommonProps';
 
 /**
   Field Actions
 */
-export default {
+class SharedActions implements CommonProperties {
```

Comment:
All of the shared files have to include the properties and functions they use from other shared files. I didn't want to throw everything together into one file so I include the necessary functions and properties in CommonProperties and then pull that into each shared file. It's not the prettiest but it gets the job done.